src/pages/Miruro/main.ts

```diff
@@ -0,0 +1,75 @@
+import { pageInterface } from '../pageInterface';
+
+export const Miruro: pageInterface = {
+  name: 'Miruro',
+  domain: ['https://www.miruro.tv', 'https://www.miruro.online'],
+  languages: ['English'],
+  type: 'anime',
+  isSyncPage(url) {
+    return Boolean(j.$('.player[data-media-player]').length);
+  },
+  isOverviewPage(url) {
+    return false;
+  },
+  getImage() {
+    return $('.player[data-media-player] .vds-poster').attr('src');
+  },
+  sync: {
+    getTitle(url) {
+      return utils.urlPart(url, 5).replace(/-/g, ' ').trim();
+    },
+    getIdentifier(url) {
+      // '139630-1' for https://www.miruro.tv/watch/139630/boku-no-hero-academia-6th-season/1
+      return `${utils.urlPart(url, 4)}-${utils.urlPart(url, 6)}`;
```

Comment:
The identifier should be per anime and not per episode

Reply:
I've made changes such that the ID of the anime is used as identifier `${utils.urlPart(url, 4)}`